subsys/lorawan/lorawan.c

```diff
@@ -7,12 +7,13 @@
 #include <init.h>
 #include <errno.h>
 #include <lorawan/lorawan.h>
-#include <zephyr.h>
 
 #include "lw_priv.h"
 
 #include <LoRaMac.h>
 #include <Region.h>
+#include "nvm/lorawan_nvm.h"
+#include <settings/settings.h>
```

Comment:
That this include is needed signals to me that the nvm backend isn't completely split out.